packages/components/src/form-token-field/style.scss

```diff
@@ -1,14 +1,10 @@
 .components-form-token-field__input-container {
 	@include input-control();
-	display: flex;
-	flex-wrap: wrap;
-	align-items: center;
 	width: 100%;
 	margin: 0 0 $grid-unit-10 0;
-	padding: $grid-unit-05*0.5 $grid-unit-05;
```

Comment:
We're refactoring this padding so it's handled by a subcontainer that doesn't contain the suggestions list (`<TokensAndInputWrapper>`). This allows us to remove the awkward negative margins on `.components-form-token-field__suggestions-list` that were needed to break out of that padding.